conf/messages.en

```diff
@@ -304,7 +304,7 @@ addressWithoutId.error.addressLine1.invalid = Address line 1 must only include l
 addressWithoutId.error.addressLine2.invalid = Address line 2 must only include letters a to z, numbers 0 to 9, apostrophes, commas, ampersands, hyphens, full stops and spaces
 addressWithoutId.error.addressLine3.invalid = City must only include letters a to z, numbers 0 to 9, apostrophes, commas, ampersands, hyphens, full stops and spaces
 addressWithoutId.error.addressLine4.invalid = Region (optional) must only include letters a to z, numbers 0 to 9, apostrophes, commas, ampersands, hyphens, full stops and spaces
-addressWithoutId.error.postcode.required = If the address is in Jersey, Guernsey or the Isle of Man, enter postcode
+addressWithoutId.error.postcode.required = Enter a postcode
```

Comment:
We still require the error: "If the address is in Jersey, Guernsey or the Isle of Man, enter postcode" for without ID (Non UK) address.